pkg/observer/observer.go

```diff
@@ -252,6 +252,8 @@ type Observer struct {
 
 	/* YAML Configuration File */
 	configFile string
+
+	Sensors []*sensors.Sensor
```

Comment:
It fees a bit strange adding this here. The way that the observer is supposed to manage sensor is via the "sensor manager" which is a goroutine for managing sensors.

See: https://github.com/cilium/tetragon/blob/bb6f985d32a3a5777d9c922961c77ac5d0bdbfb9/pkg/sensors/sync.go#L24-L30

From what I can tell, this is for testing only, so maybe we can keep the list elsewhere if we need to?

Reply:
as discussed later I'll change this to have sensors are return value